google/resource_google_project_organization_policy_test.go

```diff
@@ -0,0 +1,289 @@
+package google
+
+import (
+	"fmt"
+	"reflect"
+	"sort"
+	"testing"
+
+	"github.com/hashicorp/terraform/helper/resource"
+	"github.com/hashicorp/terraform/terraform"
+	"google.golang.org/api/cloudresourcemanager/v1"
+)
+
```

Comment:
Mind adding a comment here explaining why the tests don't run in parallel? (that way nobody tries to re-parallelize them)